### ISSUE TITLE
Would love more chapters :smile: 

### ISSUE THREAD

**Issue by CodeAsm** [natanalt/uefi-guide#1]
First issues always fun.
any plans on working on your book? Ok I have my own notes on how to write efi variables, from Linux and uefi shell, but the programming chapters peaked my interest.

And its only a table of contents :cry: 

Thanks for the awesome other repo's

**Comment by natanalt** [comment]
You've stumbled upon one of my infinite project ideas that I completely forgot about. I'll think about coming back to it eventually, and actually writing it ^^

To be honest, I have no idea how you even came across this repo lmao. I haven't promoted it anywhere, did some search engine happen to pick it up or something?

**Comment by CodeAsm** [comment]
Yes, I am so aware of the perpetual engine called our engine the brain, making projects and jumping all over the place :wink: :laughing: 

It was an search engine query indeed: https://duckduckgo.com/?q=uefi+developer+guide and the 5th result was here: https://natanalt.github.io/uefi-guide/
(wasnt my first ever try to build or learn about uefi) I started that day wondering why their build doesnt work on my Pi5 https://github.com/worproject/rpi5-uefi/blob/v0.2/README.md#getting-started and well... that simple hello world of mine isnt enough, i wondered if I could make more interesting stuff and started searching. 

Their discord was fun too, someone showed a picture of UEFI ported to the Original xbox. im like "waaat"... :thinking: (was in an emulator tho, so on real hardware it would require a modchip/tsop flash)
